_playwright-tests/Integration/CanUpdateSystemWithTemplate.spec.ts

```diff
@@ -0,0 +1,156 @@
+      if (exist?.exitCode != 0) {
+        console.log(exist?.stdout);
+        console.log(exist?.stderr);
+      }
+      expect(exist?.exitCode).toBe(0);
+      firstVersion = exist?.stdout?.toString();
+    });
+
+    await test.step('Update the template date to latest', async () => {
+      const rowTemplate = await getRowByNameOrUrl(page, templateName);
+      await rowTemplate.getByRole('button', { name: templateName }).click();
+      await expect(page.getByLabel('Breadcrumb').first()).toHaveText('RHELContentTemplates');
+      await expect(page.getByRole('heading', { level: 1 })).toHaveText(templateName);
+      await expect(page.getByText('Description:Template test')).toBeVisible();
+      await page.getByRole('button', { name: 'Actions' }).click();
+      await page.getByRole('menuitem', { name: 'Edit' }).click();
+      await expect(
+        page.getByRole('heading', { name: 'Define template content', exact: true }),
+      ).toBeVisible();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(
+        page.getByRole('heading', { name: 'Additional Red Hat repositories', exact: true }),
+      ).toBeVisible();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(
+        page.getByRole('heading', { name: 'Custom repositories', exact: true }),
+      ).toBeVisible();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByText('Use latest content', { exact: true }).click();
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await expect(page.getByText('Enter template details')).toBeVisible();
+      await expect(page.getByPlaceholder('Enter name')).toHaveValue(`${templateName}`);
+      await expect(page.getByPlaceholder('Description')).toHaveValue('Template test');
+      await page.getByPlaceholder('Description').fill('Template test edited');
+      await page.getByRole('button', { name: 'Next', exact: true }).click();
+      await page.getByRole('button', { name: 'Confirm changes', exact: true }).click();
+    });
+
+    await test.step('Update system', async () => {
+      // refresh subscription-manager
+      const subManRefresh = await client.Exec(['subscription-manager', 'refresh', '--force']);
+      expect(subManRefresh?.exitCode).toBe(0);
+
+      // clean cached metadata
+      const dnfCleanAll = await client.Exec(['dnf', 'clean', 'all']);
+      expect(dnfCleanAll?.exitCode).toBe(0);
+
+      // List errata the system is vulnerable to
+      const exist = await client.Exec([
+        'sh',
+        '-c',
+        'dnf updateinfo --list --all | grep RH | sort | tail -n 1',
+      ]);
+      if (exist?.exitCode != 0) {
+        console.log(exist?.stdout);
+        console.log(exist?.stderr);
+      }
+      expect(exist?.exitCode).toBe(0);
+      const secondVersion = exist?.stdout?.toString();
+      expect(secondVersion).not.toBe(firstVersion);
+    });
```

Comment:
Would you also be able to add a test that installs a package (something like vim-enhanced?)  This would verify that RPMs can be downloaded too

Reply:
and it wouldn't really matter which version of the template is used

Reply:
Hi @rverdile 
Downgrading a package is one way to create the need for an update. In my Satellite days, for an errata enumeration test,  I installed a package from CentOS to trigger the need for an update.

Reply:
That makes sense. In this case we would want changing the template date to make a new package version available (and cause the need for an update), so I'm not sure it would verify that, unless I'm misunderstanding.